python/tk_multi_publish2/api/tree.py

```diff
@@ -137,7 +137,7 @@ def load_file(file_path):
                 return PublishTree.load(tree_file_obj)
             except Exception, e:
                 logger.error(
-                    "Erorr trying to load publish tree from file: %s" % (e,)
+                    "Error trying to load publish tree from file: %s" % (e,)
```

Comment:
might be worth including the file path here too? Makes support easier.

Reply:
Unfortunately this code can be executed by code paths that either load from a string or from a file, so it's trickier. If you go to the load_file method we go catch and rethrow the exception in order to log.error the file name.

Reply:
But the method takes a `file_path` input argument? Would it be misleading to log that?

Reply:
Oh sorry, I misread that. We're in the right method to do this. Great, let's add it.

Reply:
🤦‍♂️